src/main/flight/pid.c

```diff
@@ -609,8 +663,31 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
         }
 
         // -----calculate I component
+        float itermErrorRate;
+        if (itermRelax) {
+            const float gyroTargetLow = pt1FilterApply(
+                &windupLpf[axis][0], currentPidSetpoint - acCorrection) + acCorrection;
+            const float gyroTargetHigh = pt1FilterApply(
+                &windupLpf[axis][1], currentPidSetpoint - acCorrection) + acCorrection;
+            DEBUG_SET(DEBUG_ITERM_RELAX, 0, gyroTargetHigh);
+            DEBUG_SET(DEBUG_ITERM_RELAX, 1, gyroTargetLow);
+            const float gmax = MAX(gyroTargetHigh, gyroTargetLow);
+            const float gmin = MIN(gyroTargetHigh, gyroTargetLow);
+            if (gyroRate >= gmin && gyroRate <= gmax) {
+                itermErrorRate = 0.0f;
+            } else {
+                itermErrorRate = (gyroRate > gmax ? gmax : gmin ) - gyroRate;
+            }
+        } else {
+            itermErrorRate = errorRate;
+        }
+        
+        if (acGain > 0) {
+            axisError[axis] = constrainf(axisError[axis] + (itermErrorRate - acCorrection) * dT, -acErrorLimit, acErrorLimit);
```

Comment:
This will add `-acCorrection` when gyro rate is in `windupLpf` window. Is that desired?

Reply:
Yeah it is.